libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Instrumentation/SFMethodInterceptor.m

```diff
@@ -232,7 +250,6 @@ - (void)swizzle {
         
         class_replaceMethod(originalMetaClass, self.originalMethodRenamedSelector, originalMethodIMP, methodTypes);
     }
-    _enabled = YES;
```

Comment:
These were unnecessary, as swizzle and unswizzle are only being called as part of the `enable` property trigger.